src/test/java/uk/gov/ons/ctp/response/action/endpoint/ActionRuleEndpointUnitTest.java

```diff
@@ -149,16 +157,17 @@ public void findActionRulesByActionPlanNoRules() throws Exception {
   @Test
   public void findActionRulesByActionPlan() throws Exception {
     final List<ActionRule> results = new ArrayList<>();
-    for (int i = 0; i < 3; i++) {
-      results.add((actionRules.get(i)));
-    }
-    when(actionRuleService.findActionRulesByActionPlanId(ACTION_PLAN_ID_1)).thenReturn(results);
+    results.add(ActionRule.builder().id(ACTION_RULE_ID_1).priority(1).actionTypeFK(1).build());
```

Comment:
👍 explicitness over less repeated code in tests every time. This helps a lot with readability